tests/manage/z_cluster/cluster_expansion/test_add_capacity.py

```diff
@@ -17,35 +13,10 @@ class TestAddCapacity(ManageTest):
     """
     Automates adding variable capacity to the cluster while IOs running
     """
-    @pytest.mark.parametrize(
-        argnames=[
-            "node_multiplier", "capacity"
-        ],
-        argvalues=[
-            pytest.param(
-                *[3, '2000Gi'],
-            ),
-        ]
-    )
-    def test_add_capacity(self, node_multiplier, capacity):
+    def test_add_capacity(self):
         """
         Test to add variable capacity to the OSD cluster while IOs running
-
-        Args:
-        node_multiplier: the number of OSD to add per worker node
-        capacity: the storage capacity of each OSD
         """
-        dt = config.ENV_DATA['deployment_type']
-        if dt == 'ipi':
-            storage_cluster = machine_utils.get_storage_cluster(namespace=defaults.ROOK_CLUSTER_NAMESPACE)
-            worker_nodes = len(get_typed_nodes())
-            machine_utils.add_capacity(storagecluster_name=storage_cluster, count=worker_nodes * node_multiplier)
-            machine_utils.add_storage_capacity(storagecluster_name=storage_cluster, capacity=capacity)
-            pod_obj = ocp.OCP(kind=constants.POD, namespace=defaults.ROOK_CLUSTER_NAMESPACE)
-            assert pod_obj.wait_for_resource(
-                condition=constants.STATUS_RUNNING, selector=constants.OSD_APP_LABEL,
-                resource_count=worker_nodes * node_multiplier, timeout=600
-            ), "OSD pods failed to reach RUNNING state"
-        else:
-            pytest.skip("UPI not yet supported")
-        # ToDo run IOs
+        osd_size = storage_cluster.get_osd_size()
```

Comment:
this works for OCS 4.3 only as expansion there is done in the increments of size of the osd with which cluster is deployed initially.

For OCS 4.2, you need to pass the capacity in the multiples of 2TiB.